semperfiwebdesign/all-in-one-seo-pack
# Change 'Media' to 'Media / Attachments'

Issue #1757

## Proposed changes

'Media' can be confusing so we're changing the post label to 'Media / Attachments' in our UI to make it more clear.

## Types of changes

What types of changes does your code introduce?

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Improves existing functionalty

## Checklist


## Testing instructions

Check all modules to make sure that we no longer use 'Media' but use 'Media / Attachments' instead (General Settings, Social Meta, XML Sitemap, Video Sitemap, Importer/Exporter). 

Also, uncheck/check the checkboxes for 'Media / Attachments' and make sure that the options are still being saved to the database.

## Further comments

None.
